python-client/giskard/push/utils.py

```diff
@@ -31,3 +31,10 @@ def bins_count(model, dataframe):  # done at the beggining
         flag = {value: count < ratio for value, count in counts.items()}
         value_counts[column] = {"value_counts": counts, "nunique": nunique, "ratio": ratio, "flag": flag}
     return value_counts
+
+
+def compute_mad(x):
```

Comment:
I would also suggest to use the scipy stats method, it's fast and more transparent. The only down side could be the version compatibility, but as long as it works with the scipy that we ship, should be fine. 
![Screenshot 2023-07-24 at 16 35 10](https://github.com/Giskard-AI/giskard/assets/32709181/6e53ca85-97bd-48ff-a3cc-2a1bd3cca18c)